Index.html

```diff
@@ -18,95 +18,208 @@
-                <ul>
-                    <li>
-                        <p>Codekit</p>
-                    </li>
-                    <li>
-                        <p>Github</p>
-                    </li>
-                    <li>
-                        <p>Codepen</p>
-                    </li>
-                    <li>
-                        <p>Gitlab</p>
-                    </li>
-                    <li>
-                        <p>Terminal</p>
-                    </li>
-                </ul>
-            </section>
-        </article>
-        <div class="vector">
-            <img src="./images/Vector4.png" alt="vector">
         </div>
-        <div class="socialContent">
-            <ul>
-                <li><a href="#"><img src="./icons/github.png" alt="github icon"></a></li>
-                <li><a href="#"><img src="./icons/linkedIn.png" alt="linkedIn icon"></a></li>
-                <li><a href="#"><img src="./icons/angelist.png" alt="angelist icon"></a></li>
-                <li><a href="#"><img src="./icons/twitter.png" alt="twitter icon"></a></li>
-                <li><a href="#"><img src="./icons/medium.png" alt="medium icon"></a></li>
-            </ul>
+        <div class="grid-item recent-work">
+            <p>MY RECENT WORK</p>
+            
+                     <div class="grid-container frame47">
+                        <div class="vector">
+                            <img src="./images/Vector4.png" alt="vector">
+                        </div>
+                <div class="grid-item page page1">
+                    <img class="imgPlace" src="./images/ImgPlaceholder.png" alt="PlaceHolder">
+                    <div class="block">
+                        <h2>Multi-Post Stories</h2>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                            industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                        </p>
+                        <ul>
+                            <li>
+                                <a href="#"><p>CSS</p></a>
+                            </li>
+                            <li>
+                                <a href="#"><p>html</p></a>
+                            </li>
+                            <li>
+                                <a href="#"><p>bootstrap</p></a>
+                            </li>
+                            <li>
+                                <a href="#"><p>Ruby</p></a>
+                            </li>
+                        </ul>
+                    
+                        <button type="submit"  >  See Project</button>
+                    </div>
+                </div>
+                <div class="grid-item page page2"> <img class="imgPlace" src="./images/ImgPlaceholder.png" alt="PlaceHolder">
+                    <div class="block">
+                        <h2>Multi-Post Stories</h2>
```

Comment:
- [ ] Kindly make every card title the `<h2>Multi-Post Stories</h2>` to the left like the design, not in the center.

![image](https://user-images.githubusercontent.com/35535130/154597763-ccec1e86-d02c-41a2-8725-0f0ebb7fd49f.png)